views/js/runner/provider/qti.js

```diff
@@ -405,10 +405,8 @@ define([
                         self.trigger('endsession');
                     }
                 })
-                .on('log', function(data){
-                    self.getProxy().callTestAction('log', {
-                        logData: data
-                    });
+                .on('traceLog', function(traceData){
```

Comment:
I wouldn't expose this event for the reasons described in https://github.com/oat-sa/extension-tao-test-runner-plugins/pull/21#pullrequestreview-60821957